resources/lang/nl/permissions.php

```diff
@@ -71,4 +71,6 @@
     'access_utility' => ':title',
     'access_utility_desc' => 'Geeft toegang tot het :title gereedschap',
     'group_misc' => 'Diversen',
+    'resolve_duplicate_ids' => 'Los dubbele ID''s op',
+    'resolve_duplicate_ids_desc' => 'Geeft de mogelijkheid om dubbele ID''s te zien en op te lossen.',
```

Comment:
The apostrophes in here cause PHP syntax errors. Did you add these manually or did the CLI tool cause them?